src/image/VisitInfo.cc

```diff
@@ -247,7 +279,8 @@ class VisitInfoFactory : public table::io::PersistableFactory {
                               coord::Observatory(record.get(keys.longitude), record.get(keys.latitude),
                                                  record.get(keys.elevation)),
                               coord::Weather(record.get(keys.airTemperature), record.get(keys.airPressure),
-                                             record.get(keys.humidity))));
+                                             record.get(keys.humidity)),
+                              record.get(keys.instrumentLabel)));
```

Comment:
This one is a bit tough: I needed to add this to the end of the constructor to support non-kwarg constructor, but it really blogs in with the "observatory" item since it's most related to that, not "weather". 

I guess the best solution is to just move it to the end of the schema definition and leave a newline after the weather.

Reply:
I'm not sure I follow. You're making a breaking change to the constructor no matter what (there are no defaults), so why does it matter where you add it to the list?

Reply:
There are defaults in the pybind11 for the python level.

Reply:
I reordered things so instrumentLabel is always at the end of any such lists.